src/components/movies/movieMostPopular.vue

```diff
@@ -0,0 +1,25 @@
+
+<template>
+  <div>
+    <moviesMix :moviesEndpoint="'popular'"></moviesMix>
+  </div>
+</template>
+
+<script>
+import moviesMix from "./moviesMix.vue";
+export default {
+  components: {
+    moviesMix: moviesMix,
```

Comment:
Component names should start in uppercase. And same do define them like this:
`
components: {
    moviesMix
}
`